development/nautobot_config.py

```diff
@@ -2,18 +2,20 @@
 
 import os
 
-from nautobot.core.settings import *  # noqa: F403  # undefined-local-with-import-star
+#
+# Debugging defaults to True rather than False for the development environment
+#
+os.environ["NAUTOBOT_DEBUG"] = os.getenv("NAUTOBOT_DEBUG", "True")
```

Comment:
What about removing this line, and adding `NAUTOBOT_DEBUG=true` to `development/dev.env` instead? @glennmatthews